lib/eslint-patch.js

```diff
@@ -1,13 +1,22 @@
 "use strict";
 
 const Plugins = require("eslint/lib/config/plugins");
+const ModuleResolver = require("eslint/lib/util/module-resolver");
+
+const ConfigFile = require("eslint/lib/config/config-file");
```

Comment:
it looks like we aren't using this variable

1. do we need to require it?
1. I'm surprised we don't have an eslint check creating issue about this unused variable (we have [this](https://github.com/codeclimate/styleguide/pull/43) in our styleguide) -- is that misconfigured maybe?